examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/README.md

```diff
@@ -0,0 +1,64 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+## Apache Spark Streaming Receiver for Pulsar
```

Comment:
Can you update the example to website - http://pulsar.apache.org/docs/en/adaptors-spark/ ?

It is more obvious to find in website than README in a github repo.

Reply:
> Can you update the example to website - http://pulsar.apache.org/docs/en/adaptors-spark/ ?
> 
> It is more obvious to find in website than README in a github repo.

done.